### PR TITLE
Fix unit tests

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

**Comment by rosen-vladimirov** [comment]
edit-configuration test: doesn't modify file if correct configuration file is given and it exists:
fails on Linux

File: lib/hooks/typescript-compilation.ts [hunk 1]
```diff
@@ -4,13 +4,13 @@ import path = require("path");
 
 require("./../bootstrap");
 import fiberBootstrap = require("./../fiber-bootstrap");
-import commonHelpers = require("./../common/helpers");
 fiberBootstrap.run(() => {
 	$injector.require("typeScriptCompilationService", "./common/services/typescript-compilation-service");
 
 	var project: Project.IProject = $injector.resolve("project");
+	var fs: IFileSystem = $injector.resolve("fs");
```
Comment (teobugslayer) [inline lib/hooks/typescript-compilation.ts]: I would name this $fs

**Comment by teobugslayer** [comment]
:+1: aside from a minor comment

**Comment by Fatme** [comment]
run ci

**Comment by rosen-vladimirov** [comment]
:+1:  after green build

**Comment by rosen-vladimirov** [comment]
merge :100: 
Great work

**Pull request merged**